test/integration/data-types.test.js

```diff
@@ -274,3 +274,30 @@ describe(Support.getTestDialectTeaser('DataTypes'), function() {
     testFailure(Type);
   });
 });
+
+describe(Support.getTestDialectTeaser('DataTypes') + " MySQL", function() {
```

Comment:
`Support.getTestDialectTeaser` already appends mysql.
Would it be possible to simply have a general empty coordinates test for all dialects?